puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-plugins-pull-request.yaml

```diff
@@ -31,15 +31,14 @@
           values:
             - postgresql
             - sqlite3
-            - mysql
```

Comment:
I think this needs to stay in if we want to keep testing it for stable jobs.

Reply:
ok, i reverted the change to this file

Reply:
Looking back at it, it's easier to just drop mysql altogether. The combination filters will just be very complex.